packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -205,22 +206,11 @@ export class ExecutionEngineHttp implements IExecutionEngine {
       };
     }
 
-    const {status, latestValidHash, validationError} = await (
-      this.rpcFetchQueue.push(engineRequest) as Promise<EngineApiRpcReturnTypes[typeof method]>
-    )
-      // If there are errors by EL like connection refused, internal error, they need to be
-      // treated separate from being INVALID. For now, just pass the error upstream.
-      .catch((e: Error): EngineApiRpcReturnTypes[typeof method] => {
```

Comment:
this error handling is required upstream in processBlocks (as well as to not downgrade the peer in case of UNAVAILABLE)